### PR TITLE
Fix deposit type emitted from contract.

### PR DESCRIPTION
**Pull request by Avi-D-coder**
This will be switched to a pr against main after `feature/contract-submit-batch` is merged.

### CODE REVIEW

**Comment by Avi-D-coder** [comment]
This pr is blocked on the cli implementation

File: kairos-session-code/deposit-session/src/main.rs [hunk 1]
```diff
@@ -30,6 +31,7 @@ pub extern "C" fn call() {
         "deposit",
         runtime_args! {
             "temp_purse" => temp_purse,
+            "recipient" => public_key,
```
Comment (jonas089) [inline kairos-session-code/deposit-session/src/main.rs:34]: Should this really be a Casper PublicKey? Shouldn't this be a Kairos PublicKey?

Comment (jonas089) [reply kairos-session-code/deposit-session/src/main.rs:34]: Other than that lgtm.

Comment (Avi-D-coder) [reply kairos-session-code/deposit-session/src/main.rs:34]: Kairos PublicKey is just Vec<u8> which would need CLType::Any, so I convert to bytes in the contract instead.

**Comment by Avi-D-coder** [comment]
The nixos test is still broken, but please review.

**Review by marijanp — APPROVED** [review]

**Pull request merged**